Fix formatting for `if` clauses in `match-case` blocks (#4269)
psf/black: CHANGES.md

<!-- Changes that affect Black's preview style -->

> - Fixed a bug where `if` clauses in `match-case` blocks are not wrapped with parenthesis
>   when the line is too long. (#4269)

Reviewer: ```suggestion
- `if` guards in `case` blocks are now wrapped in parentheses
  when the line is too long. (#4269)
```
